eva-server/src/main/java/uk/ac/ebi/eva/server/ws/GeneWSServerV2.java

```diff
@@ -59,13 +60,23 @@ public GeneWSServerV2() {
     }
 
     @GetMapping(value = "/{geneIds}/variants")
-    public ResponseEntity getVariantsByGene(@PathVariable("geneIds") List<String> geneIds,
-                                            @RequestParam(name = "species") String species,
-                                            @RequestParam(name = "assembly") String assembly,
-                                            @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                            @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                            HttpServletResponse response,
-                                            @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByGene(
+            @ApiParam(value = "Comma separated values of gene names or gene ids. e.g. BRCA2,FOXP2,ENSG00000223972")
```

Comment:
```suggestion
            @ApiParam(value = "Comma separated gene symbols and/or Ensembl gene IDs, e.g. BRCA2,FOXP2,ENSG00000223972")
```
I also suggest one of these is made an example for illustration purposes of a default search.